tests/dflowfm/test_extold.py

```diff
@@ -0,0 +1,772 @@
+            "",
+            "QUANTITY=waterlevelbnd",
+            "FILENAME=OB_001_orgsize.pli",
+            "FILETYPE=9",
+            "METHOD=3",
+            "OPERAND=O",
+        ]
+
+        forcing_1 = ExtForcing(
+            quantity=Quantity.InternalTidesFrictionCoefficient,
+            filename=Path("surroundingDomain.pol"),
+            filetype=FileType.NetCDFGridData,
+            method=Method.InterpolateSpace,
+            operand=Operand.SuperimposeNewValues,
+            value=0.0125,
+        )
+
+        forcing_2 = ExtForcing(
+            quantity=Quantity.WaterLevelBnd,
+            filename=Path("OB_001_orgsize.pli"),
+            filetype=FileType.Polyline,
+            method=Method.InterpolateTimeAndSpaceSaveWeights,
+            operand=Operand.OverwriteExistingValues,
+        )
+
+        model = ExtOldModel(forcing=[forcing_1, forcing_2])
+
+        model.serializer_config.float_format = "f"
+
+        with get_temp_file("test_save_model.ext") as file:
+            model.save(filepath=file)
+
+            with create_temp_file_from_lines(
+                exp_file_content, "test_save_model_expected.ext"
+            ) as exp_file:
+                assert_files_equal(file, exp_file)
+
+
+class TestParser:
+    def test_parse_two_blocks_parses_to_the_correct_dictionaries(self):
+        file_content = [
+            "* This is a comment",
+            "* This is a comment",
+            "",
+            "QUANTITY=internaltidesfrictioncoefficient",
+            "FILENAME=surroundingDomain.pol",
+            "FILETYPE=11",
+            "METHOD=4",
+            "OPERAND=+",
+            "VALUE=0.0125",
+            "",
+            "* This is a comment",
+            "",
+            "QUANTITY=waterlevelbnd",
+            "FILENAME=OB_001_orgsize.pli",
+            "FILETYPE=9",
+            "METHOD=3",
+            "* This is a comment",
+            "OPERAND=O",
+            "* This is a comment",
+        ]
+
+        parser = Parser()
+
+        with create_temp_file_from_lines(file_content, "two_blocks.ext") as temp_file:
+            data = parser.parse(filepath=temp_file)
+
+        assert len(data) == 1
+        forcing_list = data["forcing"]
+
+        assert len(forcing_list) == 2
+
+        forcing_1 = forcing_list[0]
+        assert len(forcing_1) == 6
+
+        assert forcing_1["QUANTITY"] == "internaltidesfrictioncoefficient"
+        assert forcing_1["FILENAME"] == "surroundingDomain.pol"
+        assert forcing_1["FILETYPE"] == "11"
+        assert forcing_1["METHOD"] == "4"
+        assert forcing_1["OPERAND"] == "+"
+        assert forcing_1["VALUE"] == "0.0125"
+
+        forcing_2 = forcing_list[1]
+        assert len(forcing_2) == 5
+
+        assert forcing_2["QUANTITY"] == "waterlevelbnd"
+        assert forcing_2["FILENAME"] == "OB_001_orgsize.pli"
+        assert forcing_2["FILETYPE"] == "9"
+        assert forcing_2["METHOD"] == "3"
+        assert forcing_2["OPERAND"] == "O"
```

Comment:
Nothing wrong with the test, but I wonder if it is more readable if you do it like this:
```
expected_data = {
            "forcing": [
                {
                    "QUANTITY": "internaltidesfrictioncoefficient",
                    "FILENAME": "surroundingDomain.pol",
                    "FILETYPE": "11",
                    "METHOD": "4",
                    "OPERAND": "+",
                    "VALUE": "0.0125",
                },
                {
                    "QUANTITY": "waterlevelbnd",
                    "FILENAME": "OB_001_orgsize.pli",
                    "FILETYPE": "9",
                    "METHOD": "3",
                    "OPERAND": "O",
                },
            ]
        }

        assert data == expected_data
```
What do you think?

Reply:
That is indeed way better.